### ISSUE TITLE
Review GA results

### ISSUE THREAD

**Issue by j08lue** [NASA-IMPACT/veda-config-ghg#202]
Some page titles apparently contain "VEDA UI", according to our Google Analytics dashboard.

@aboydnw can provide access to the GA (Looker Studio) dashboard

## Acceptance criteria

- [x] Checked and removed title, if in fact an issue

**Comment by aboydnw** [comment]
I think it might have more to do with our GA configuration. Slesa mentioned that she added staging to the data stream. I will have to investigate how we can clean it up. When I create a table of all the page titles including VEDA in our GHG data stream, they are all localhost page URLs. All of the ghg.center URLs look clean. I will keep investigating.
![image](https://github.com/NASA-IMPACT/veda-config-ghg/assets/92742765/f37203d6-a9a1-4a38-ae2f-e81f9347a6e2)

**Comment by aboydnw** [comment]
Okay I am still investigating this. But for now I just filtered out these values from the dashboard so this problem isn't as immediate. However, I found something weird when looking through the data. I see records with the same page URL but different titles. I am not sure if this is something to do with google analytics or if it's something in the code. 

For example, here are all the different entries with url of the analysis page.
![image](https://github.com/NASA-IMPACT/veda-config-ghg/assets/92742765/c9f0b3f6-9225-4a3e-a88d-24b3b6ee813a)

And then when I look at all the entries I see lots of duplicates. And the number of visits to the analysis page url with the simple VEDA Dashboard title matches what I would expect from our home page (relative to all the other pages)
![image](https://github.com/NASA-IMPACT/veda-config-ghg/assets/92742765/e815240f-89ec-4de9-8b0d-cb4fe7253f31)

Any ideas on why this might be? Any way we could verify there's not something weird going on in the dashboard? I did a quick google for any google analytics related issues and nothing jumped out at me
